flink-connectors/flink-connector-elasticsearch6/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/table/Elasticsearch6DynamicSinkFactory.java

```diff
@@ -149,6 +151,15 @@ private void validate(Elasticsearch6Configuration config, Configuration original
 					config.getPassword().orElse("")
 				));
 		}
+		if (config.getParellelism().isPresent()) {
```

Comment:
The parallelism has been validated in `CommonPhysicalSink`,so i think it's unnecessary here.

Reply:
ok

Reply:
> The description about the PR should be what changes you do.

ok,I will add description when it's ready for review